### DISCUSSION TITLE
Youtube is not working

### DISCUSSION THREAD

**Discussion by smsaq4u** [Alex313031/Thorium-Win#109] [Q&A] [answered]
![Youtube not working](https://github.com/Alex313031/Thorium-Win/assets/149236784/e447e1f1-8761-4482-b9be-0eba48ef820f)

**Youtube is not working**

**Comment by gz83** [comment]
The prompt on the youtube page has told you the specific reason, which has nothing to do with the Thorium browser.

**Reply by smsaq4u** [reply]
it was working OK , also BRAVE is working OK after updating components, Thorium working OK with youtube, but not now..!!

**Reply by smsaq4u** [reply]
Youtube works great in Thorium Browser incognito Mode..!!

**Reply by gz83** [reply, marked as answer]
I have made it very clear before that the problem lies in the ad blocking extension. Just turn off the ad blocking extension and YouTube will work normally. If you are not satisfied with this change, it is recommended to give feedback to YouTube.
